### PR TITLE
Fixes admin priviledge to stop work in activity

### LINKED ISSUE #2165: No 'Stop Work' / 'Stop Work & Ding Rep'
## User Story
As an 'Admin', I'd like the ability to 'Stop Work' / 'Stop Work & Ding Rep' from the front-end. This was a feature available to me to easily remove stale workers from bounties. 

This is no longer available. I'd like this feature back so I can ding rep appropriately when hunters disappear. 

## Timeline
Triage - P2 (Minor Bug): A feature of Gitcoin fails to follow expected behavior. The issue is not high magnitude. 

screen shot 2018-09-05 at 3 12 17 pm

### PR DESCRIPTION
**Pull request by SaptakS**
##### Description
 ensure that when a worker has applied -> funder/admin can ONLY approve / reject.
Upon being approved -> funder/admin would be able to stop work / stop work + ding repo

##### Checklist

- [x] linter status: 100% pass
- [x] changes don't break existing behavior
- [x] commit message follows [commit guidelines](https://github.com/gitcoinco/web/blob/master/docs/CONTRIBUTING.md#step-4-commit)

##### Refers/Fixes

Fixes #2165

### CODE REVIEW

**Comment by SaptakS** [comment]
Let's get this tested once on staging.

**Comment by vs77bb** [comment]
Happy to review when this is on staging, just lmk!

**Comment by thelostone-mc** [comment]
@SaptakS wanna rebase and get it on staging ?

File: app/dashboard/templates/bounty/details.html [hunk 1]
```diff
@@ -311,6 +303,23 @@ <h5 class="bounty-heading">{% trans "Funder" %}</h5>
...
+                    <a id="remove-[[:name]]" class="button button--primary" role="button" href="remove-handle">
+                      <span class="font-smaller-4">Stop work</span>
+                    </a>
+                  </span>
+                </div>
+                [[if slash_possible]]
+                  <div class="stop-work">
+                    <span title="<div class='tooltip-info tooltip-sm'>Remove this user's started work on the bounty and ding rep</div>">
+                      <a id="remove-slash-[[:name]]" class="button button--primary admin-only" role="button" href="remove-handle">
+                        <span class="font-smaller-4">Stop work + Ding Rep</span>
```
Comment (mbeacom) [inline app/dashboard/templates/bounty/details.html]: One thing here...  should we modify the newly introduced copy to support i18n?

**Pull request merged**